packages/contracts-bedrock/deploy-config/mainnet.json

```diff
@@ -14,11 +14,11 @@
   "batchInboxAddress": "0xff00000000000000000000000000000000000010",
   "batchSenderAddress": "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
   "l2OutputOracleSubmissionInterval": 1800,
-  "l2OutputOracleStartingTimestamp": 1683643307,
-  "l2OutputOracleStartingBlockNumber": 79149146,
+  "l2OutputOracleStartingTimestamp": 0,
+  "l2OutputOracleStartingBlockNumber": 0,
   "l2OutputOracleProposer": "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC",
   "l2OutputOracleChallenger": "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266",
-  "finalizationPeriodSeconds": 1,
+  "finalizationPeriodSeconds": 3600,
```

Comment:
I increased this in a separate commit. Can drop back to 1 if preferred.

Feel free to push to this branch.